tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/WcfTests.cs

```diff
@@ -70,6 +70,7 @@ public static IEnumerable<object[]> GetData()
         public async Task SubmitsTraces(string metadataSchemaVersion, string binding, bool enableNewWcfInstrumentation, bool enableWcfObfuscation)
         {
             SetEnvironmentVariable("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", metadataSchemaVersion);
+            SetEnvironmentVariable("DD_TRACE_OTEL_ENABLED", "true");
```

Comment:
Is this necessary?

Reply:
Yes as I create a custom span using activities as it's done now